packages/graphql_client/lib/models/farms.dart

```diff
@@ -1,31 +1,439 @@
+  String? stellarAddressNotContains;
+  String? stellarAddressContainsInsensitive;
+  String? stellarAddressNotContainsInsensitive;
+  String? stellarAddressStartsWith;
+  String? stellarAddressNotStartsWith;
+  String? stellarAddressEndsWith;
+  String? stellarAddressNotEndsWith;
+  bool? dedicatedFarmIsNull;
+  bool? dedicatedFarmEq;
+  bool? dedicatedFarmNotEq;
+  int? limit;
+  int? offset;
+  OrderByOptions? orderBy;
+
+  FarmsQueryOptions({
+    this.idEq,
+    this.idGt,
+    this.idGte,
+    this.idNotEq,
+    this.idLt,
+    this.idLte,
+    this.idContains,
+    this.idNotContains,
+    this.idContainsInsensitive,
+    this.idNotContainsInsensitive,
+    this.idStartsWith,
+    this.idNotStartsWith,
+    this.idEndsWith,
+    this.idNotEndsWith,
+    this.gridVersionIsNull,
+    this.gridVersionEq,
+    this.gridVersionNotEq,
+    this.gridVersionGt,
+    this.gridVersionGte,
+    this.gridVersionLt,
+    this.gridVersionLte,
+    this.gridVersionIn,
+    this.gridVersionNotIn,
+    this.farmIDIsNull,
+    this.farmIDEq,
+    this.farmIDNotEq,
+    this.farmIDGt,
+    this.farmIDLte,
+    this.farmIDIn,
+    this.farmIDNotIn,
+    this.nameIsNull,
+    this.nameEq,
+    this.nameNotEq,
+    this.nameGt,
+    this.nameGte,
+    this.nameLt,
+    this.nameLte,
+    this.nameIn,
+    this.nameNotIn,
+    this.nameContains,
+    this.nameNotContains,
+    this.nameContainsInsensitive,
+    this.nameNotContainsInsensitive,
+    this.nameStartsWith,
+    this.nameNotStartsWith,
+    this.nameEndsWith,
+    this.nameNotEndsWith,
+    this.twinIDIsNull,
+    this.twinIDEq,
+    this.twinIDNotEq,
+    this.twinIDGt,
+    this.twinIDGte,
+    this.twinIDLt,
+    this.twinIDLte,
+    this.twinIDIn,
+    this.twinIDNotIn,
+    this.pricingPolicyIDIsNull,
+    this.pricingPolicyIDEq,
+    this.pricingPolicyIDNotEq,
+    this.pricingPolicyIDGt,
+    this.pricingPolicyIDGte,
+    this.pricingPolicyIDLt,
+    this.pricingPolicyIDLte,
+    this.pricingPolicyIDIn,
+    this.pricingPolicyIDNotIn,
+    this.certificationIsNull,
+    this.certificationEq,
+    this.certificationNotEq,
+    this.certificationIn,
+    this.certificationNotIn,
+    this.stellarAddressIsNull,
+    this.stellarAddressEq,
+    this.stellarAddressNotEq,
+    this.stellarAddressGt,
+    this.stellarAddressGte,
+    this.stellarAddressLt,
+    this.stellarAddressLte,
+    this.stellarAddressIn,
+    this.stellarAddressNotIn,
+    this.stellarAddressContains,
+    this.stellarAddressNotContains,
+    this.stellarAddressContainsInsensitive,
+    this.stellarAddressNotContainsInsensitive,
+    this.stellarAddressStartsWith,
+    this.stellarAddressNotStartsWith,
+    this.stellarAddressEndsWith,
+    this.stellarAddressNotEndsWith,
+    this.dedicatedFarmIsNull,
+    this.dedicatedFarmEq,
+    this.dedicatedFarmNotEq,
+    this.limit,
+    this.offset,
+    this.orderBy,
+  });
+
+  @override
+  String toString() {
+    List<String> queryOptions = [];
+
```

Comment:
- mapping to graphql not consistent 
- for example this `    addToQueryList(queryOptions, "farm_id_eq", farmIDEq);`, it should be `farmID_eq`.
- please re-check all fields against graphql